src/lando/api/views.py

```diff
@@ -303,3 +303,43 @@ def get(self, request: WSGIRequest, repo_name: str, number: int) -> JsonResponse
             target_repo, pull_request, request
         )
         return JsonResponse(warnings_and_blockers)
+
+
+class PullRequestTryPushAPIView(APIView):
+    @method_decorator(require_authenticated_user)
+    def post(
+        self, request: WSGIRequest, repo_name: str, pull_number: int
+    ) -> JsonResponse:
+        try:
+            try_repo = Repo.objects.get(name="try")
+        except Repo.DoesNotExist:
+            return JsonResponse({"errors": ["Try repo does not exist"]}, 500)
+
+        target_repo = Repo.objects.get(name=repo_name)
+        client = GitHubAPIClient(target_repo.url)
+        ldap_username = request.user.email
+        pull_request = client.build_pull_request(pull_number)
+
+        job = LandingJob.objects.create(
+            target_repo=target_repo,
+            is_handed_over=False,
+            handover_repo=try_repo,
+            requester_email=ldap_username,
+        )
+        author_name, author_email = pull_request.author
+        patch_data = {
+            "author_name": author_name,
+            "author_email": author_email,
+            "commit_message": pull_request.commit_message,
+            "timestamp": int(datetime.now().timestamp()),
```

Comment:
You could set the timestamp to the pull requests's `updatedAt` field. As far as I have seen, it gets updaded reliably with any comments and other metadata changes, in addition to plain code changes.